docs/blockchain-basics/introduction.md

```diff
@@ -15,6 +15,42 @@ import NotificationBar from '../../src/components/docs/NotificationBar';
+
+From there, countless reflexions on ethics, researchs on technologies, and of course Cryptography take place. Mails traceability (1981); Elliptic Curve Cryptography (1985); Digicash (1989); PGP (1991); CyberCash (1994); E-gold (1996); HashCash (1997, with Proof-of-Work); Smart Contracts (1997, defined by Nick Szabo). Even _Bit Gold_ and _B-money_ (from Wei Dai) are from 1998, and the latter already used a decentralized database recording exchanges and using an older version of the Bitcoin's _Proof-of-Work_ (more details in next chapters on this matter).
+
+In 2001, the Bittorent client gave more support for the Peer-2-Peer network trials and researchs, opening the way for more distribution and the decentralisation looked for.
+
+Finally, before Lehman bankruptcy and the financial crisis (2008), Hal Finney (1956-2014), developper on PGP (1991), released his "_Reusable Proof-of-Work_" (2004), before being one of the first exchanging bitcoins with Satoshi Nakamoto.
+
+From 2009, many projects copy Bitcoin, or try to enhance its capabilities: In 2011, Vitalik Buterin discovers Bitcoin and two years later (October, 2013), works on [a proposal for _Mastercoin_](https://web.archive.org/web/20150627031414/http://vbuterin.com/ultimatescripting.html), that would have upgrade the programmable side of the system, compared to Bitcoin, because bitcoins exchanges have scripts, embedded in _transactions_, that set the way funds can be spent. The idea is going beyond the main use-case of a currency.
+
+But the Mastercoin team didn't want to go in this direction, so Vitalik began to re-work his proposal into another project named _Ethereum_ (December, 2013). The need for even more complex scripts was driven by the interest into _**D**ecentralized **A**utonomous **O**rganizations_ (DAO), that other systems couldn't allow. That is also around December, 2013, that Gavin Wood (protocol-side) and Jeffrey Wilcke (client-side) join Vitalik Buterin to work on Ethereum. The white paper is published, and in January, 2014, Ethereum is announced.
+
+Follow the Yellow Paper from Gavin Wood in April, 2014, a crowdfunding in bitcoins during the same summer, and the launch on 30th July, 2015, the next year. Ethereum is now a new system, with its own cryptocurrency called "_Ether_".
+
+The first DAO of Ethereum called "_The DAO_" is created and more than 150 millions of dollars (\$) are  fundraised in June, 2016. But the 16th, 50 millions (\$) are stolen through the famous "_The DAO hack_".
```

Comment:
These needs reference [1], [2], [3],.... 
See chapters in the DeFi module to copy/paste the references format

Reply:
Do you mean links to articles?

Reply:
Yes, articles, official documentation or website or any other reference